src/mme-app/procedureStats.cpp

```diff
@@ -60,3 +60,5 @@ int ProcedureStats::num_of_ddn_received = 0;
 int ProcedureStats::num_of_service_request_received = 0;
 int ProcedureStats::num_of_ddn_ack_sent = 0;
 int ProcedureStats::num_of_tau_response_to_ue_sent = 0;
+int ProcedureStats::num_of_service_reject_sent = 0;
```

Comment:
This needs cleanup. C++ can give better design idea to increment stats...Before it become worst. Can you open JIRA ticket or github issue.

Reply:
Created task for this. Will pick this next.